elasticsearch/pom.xml

```diff
@@ -314,6 +313,7 @@
                                 <exclude>**/FSSlaveAndMasterDPTest.java</exclude>
                                 <exclude>**/IndexReaderSeeOptimizedIndexTest.java</exclude>
                             </excludes>
+                            <excludedGroups>org.hibernate.search.testsupport.junit.SkipOnElasticsearch,org.hibernate.search.testsupport.junit.ElasticsearchSupportInProgress</excludedGroups>
```

Comment:
+1 for this approach, but could we have it separated from the "improve Sort support" issue?